tensorflow/python/saved_model/builder_impl.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, export_dir):
       file_io.recursive_create_dir(self._export_dir)
 
     # Boolean to track whether variables and assets corresponding to the
-    # SavedModel have been saved. Specifically, the first meta graph to be added
+    # SavedModel has been saved. Specifically, the first meta graph to be added
```

Comment:
This should be "have" ("variables and assets [..] have")

Reply:
Made requested changes